pages/common/fold.md

```diff
@@ -0,0 +1,15 @@
+# fold
+
+> Wraps each line in an input file to fit a specified width.
```

Comment:
We should also mention that it outputs to the standard output. Initially I thought that it changes the file in-place.